examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -14,58 +14,56 @@
   },
```

Comment:
<p>I think this plot should <em>not</em> be a line plot but a dot plot, try using </p><p> </p><pre class="ql-syntax" spellcheck="false">trace.sample_stats["tree_depth"].plot(row="chain", ls="none", marker=".", alpha=.3)
</pre><p> </p><p>which should replace the whole cell. Also, style comment only, using <code>hue="chain"</code>  might look better</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>